### DISCUSSION TITLE
SVG doesn't show in load

### DISCUSSION THREAD

**Discussion by robin59161** [GrapesJS/grapesjs#3863] [Q&A]
![image](https://user-images.githubusercontent.com/73250776/137332539-95eaa5bc-4105-47d4-9a63-14ef8a771fe6.png)

My svg doesn't show on load of page, but the block 'svg' is here. I don't achieve to make it visible.

**Comment by artf** [comment]
Fixed already here  https://github.com/artf/grapesjs/issues/3866

**Reply by robin59161** [reply]
Hello, thanks a lot
